zkevm-circuits/src/tx_circuit.rs

```diff
@@ -42,7 +43,7 @@ use crate::{
 use bus_mapping::circuit_input_builder::keccak_inputs_sign_verify;
 use eth_types::{
     geth_types::{
-        TxType,
+        access_list_address_and_storage_key_sizes, TxType,
```

Comment:
I think it will make sense to rename this to: `access_list_size` and the docs on the method can explain what the tuple return value stands for, instead of having such a long function name.

Reply:
Rename function `access_list_address_and_storage_key_sizes` to `access_list_size` in commit [1988777](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/198877743211fbaef748891523986f5ace1c3293).